integration-cli/utils.go

```diff
@@ -62,6 +63,19 @@ func runCommandWithOutput(cmd *exec.Cmd) (output string, exitCode int, err error
 	return
 }
 
+func runCommandWithOutputWriteToFile(cmd *exec.Cmd, outputfile *os.File) (exitCode int, err error) {
```

Comment:
This func probably isn't needed since I think you can just do:
cmd.Stdout = ...writer to file...